### PR TITLE
PM-1729 Improve whitelist error response to include submitter id

### PR DESCRIPTION
**Pull request by piotr-iohk**
This pr improves error message when submitter is not on the whitelist. Was:
```
{"error":"Submitter is not registered"}
```
is: 
```
{"error":"Submitter is not registered: B62qnZEWR4eGpRCzzpVRmEqA8wGNTPjKmm3NSzGNzxDww9yHHo9vnRdmc"}
```
Also in the integration tests we no longer use AWS Keyspaces. (For the moment just commenting out relevant code in the integration tests).

### CODE REVIEW

File: src/integration_tests/integration_test.go [hunk 1]
```diff
@@ -14,19 +13,20 @@ func init() {
 		log.Fatalf("Failed to decode uptime service configuration: %v", err)
 	}
 
-	// Setup AWS Keyspaces database
 	setAppConfig("all")
 	config := getAppConfig()
```
Comment (simisimis) [inline src/integration_tests/integration_test.go:17]: thought: if this is not part of keyspaces, perhaps it should be moved above `// Setup AWS Keyspaces database`?

**Review by simisimis — APPROVED** [review]
other than non blocking comment looks good

File: src/integration_tests/integration_test.go (hunk 1 above)
Comment (piotr-iohk) [reply src/integration_tests/integration_test.go:17]: Indeed it is above:
```
	setAppConfig("all")
	config := getAppConfig()

	// Setup AWS Keyspaces database
       ...
```

Comment (simisimis) [reply src/integration_tests/integration_test.go:17]: :facepalm: sorry for the noise

**Pull request merged**